diff --git a/browser/components/translations/content/fullPageTranslationsPanel.inc.xhtml b/browser/components/translations/content/fullPageTranslationsPanel.inc.xhtml
--- a/browser/components/translations/content/fullPageTranslationsPanel.inc.xhtml
+++ b/browser/components/translations/content/fullPageTranslationsPanel.inc.xhtml
@@ -31,9 +31,8 @@
         <html:div id="full-page-translations-panel-intro">
           <html:span data-l10n-id="translations-panel-intro-description"></html:span>
           <html:a id="full-page-translations-panel-intro-learn-more-link"
-                  is="moz-support-link"
                   data-l10n-id="translations-panel-learn-more-link"
-                  support-page="website-translation" />
+                  href="about:astra-translations" />
         </html:div>
         <vbox id="full-page-translations-panel-lang-selection">
           <label data-l10n-id="translations-panel-from-label" id="full-page-translations-panel-from-label"></label>
@@ -108,9 +107,8 @@
         <html:div>
           <html:span id="full-page-translations-panel-error-unsupported-hint"></html:span>
           <html:a id="full-page-translations-panel-unsupported-learn-more-link"
-                  is="moz-support-link"
                   data-l10n-id="translations-panel-learn-more-link"
-                  support-page="website-translation" />
+                  href="about:astra-translations" />
         </html:div>
       </vbox>
 
