diff --git a/browser/components/translations/content/fullPageTranslationsPanel.js b/browser/components/translations/content/fullPageTranslationsPanel.js
--- a/browser/components/translations/content/fullPageTranslationsPanel.js
+++ b/browser/components/translations/content/fullPageTranslationsPanel.js
@@ -965,7 +965,7 @@
       gBrowser.selectedBrowser.browsingContext.top.embedderElement
         .documentGlobal;
     window.openTrustedLinkIn(
-      "https://support.mozilla.org/kb/website-translation",
+      "about:astra-translations",
       "tab",
       {
         forceForeground: true,
