diff --git a/toolkit/mozapps/extensions/content/aboutaddons.js b/toolkit/mozapps/extensions/content/aboutaddons.js
--- a/toolkit/mozapps/extensions/content/aboutaddons.js
+++ b/toolkit/mozapps/extensions/content/aboutaddons.js
@@ -1493,6 +1493,9 @@ class SidebarCategories extends HTMLElement {
       createLinkElement: () => {
-        let link = document.createElement("a", { is: "moz-support-link" });
-        link.setAttribute("support-page", "addons-help");
+        let link = document.createElement("a");
+        link.href =
+          "https://github.com/Hrishikeshmind/astradesktop/tree/addons-help";
+        link.target = "_blank";
+        link.rel = "noopener";
         link.id = "help-button";
         return link;
       },
