diff --git a/browser/base/content/utilityOverlay.js b/browser/base/content/utilityOverlay.js
--- a/browser/base/content/utilityOverlay.js
+++ b/browser/base/content/utilityOverlay.js
@@ -482,13 +482,7 @@
  * Appends UTM parameters to then opens the SUMO URL for device migration.
  */
 function openSwitchingDevicesPage() {
-  let url = getHelpLinkURL("switching-devices");
-  let parsedUrl = new URL(url);
-  parsedUrl.searchParams.set("utm_content", "switch-to-new-device");
-  parsedUrl.searchParams.set("utm_source", "help-menu");
-  parsedUrl.searchParams.set("utm_medium", "firefox-desktop");
-  parsedUrl.searchParams.set("utm_campaign", "migration");
-  openTrustedLinkIn(parsedUrl.href, "tab");
+  openTrustedLinkIn("about:astra-sync-devices", "tab");
 }
 
 function buildHelpMenu() {
