diff --git a/browser/components/urlbar/QuickActionsLoaderDefault.sys.mjs b/browser/components/urlbar/QuickActionsLoaderDefault.sys.mjs
index b99a80b..0bd3851 100644
--- a/browser/components/urlbar/QuickActionsLoaderDefault.sys.mjs
+++ b/browser/components/urlbar/QuickActionsLoaderDefault.sys.mjs
@@ -123,7 +123,7 @@ const DEFAULT_ACTIONS = {
     icon: "chrome://global/skin/icons/help.svg",
     label: "quickactions-help",
     onPick: openUrlFun(
-      "https://support.mozilla.org/products/firefox?as=u&utm_source=inproduct"
+      "https://github.com/Hrishikeshmind/astradesktop/issues/new?labels=bug"
     ),
   },
   firefoxview: {
