diff --git a/browser/components/customizableui/CustomizableWidgets.sys.mjs b/browser/components/customizableui/CustomizableWidgets.sys.mjs
--- a/browser/components/customizableui/CustomizableWidgets.sys.mjs
+++ b/browser/components/customizableui/CustomizableWidgets.sys.mjs
@@ -283,7 +283,6 @@
   {
     id: "open-file-button",
     l10nId: "toolbar-button-open-file",
-    shortcutId: "openFileKb",
     onCreated(aNode) {
       aNode.setAttribute("command", "Browser:OpenFile");
     },
