diff --git a/browser/locales/en-US/browser/browser.ftl b/browser/locales/en-US/browser/browser.ftl
index f05045a..f762df7 100644
--- a/browser/locales/en-US/browser/browser.ftl
+++ b/browser/locales/en-US/browser/browser.ftl
@@ -375,7 +375,7 @@ quickactions-viewsource2 = View page source
 quickactions-cmd-viewsource2 = view source, source, page source
 
 # Opens about:preferences:experimental (Firefox Labs)
-quickactions-labs = Open { -firefoxlabs-brand-name }
+quickactions-labs = Open { -brand-short-name } Labs
 quickactions-cmd-labs = labs, experiment
 
 # Tooltip text for the help button shown in the result.
